test/command/suite/select/function/snippet/delimiter_regexp/overlap.expected

```diff
@@ -0,0 +1,35 @@
+table_create Entries TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Entries content COLUMN_SCALAR ShortText
+[[0,0.0,0.0],true]
+load --table Entries
+[
+{"content": "aaa. bbb ccc AAA. BBB CCC xxx yyy. zzz."}
+]
+[[0,0.0,0.0],1]
+select Entries   --output_columns '   snippet(content,   "ccc", "[", "]",   {"width": 40,    "delimiter_regexp": "\\\\."})'
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  [
+    [
+      [
+        1
+      ],
+      [
+        [
+          "snippet",
+          null
+        ]
+      ],
+      [
+        [
+          " bbb [ccc] AAA. BBB [CCC] xxx yyy"
```

Comment:
Delimiters included in the range of  `match_width` do not seem to be recognized as a delimiter, is this intended behavior?

Reply:
Good catch!
It's intentional for me.
But the user who has this use case wants to be delimited. So I've changed the behavior.